### PR TITLE
60/my account

### PR DESCRIPTION
**Pull request by qiqqq**

### CODE REVIEW

File: components/molecules/m-update-password.vue [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+  }
+  &__button {
+    width: 100%;
+    @include for-desktop {
+      width: auto;
+    }
+  }
+}
+.message,
+.notice {
```
Comment (psmyrek) [inline components/molecules/m-update-password.vue]: `.notice` class is not used in template

File: components/molecules/m-update-password.vue [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+}
+.notice {
+  margin: $spacer-big 0 0 0;
+  font-size: $font-size-extra-small-mobile;
+  @include for-desktop {
+    max-width: 70%;
+    margin: $spacer 0 0 0;
+    font-size: $font-size-extra-small-desktop;
+  }
+}
```
Comment (psmyrek) [inline components/molecules/m-update-password.vue]: `.notice` class is not used in template

File: components/molecules/m-update-personal-data.vue [hunk 3]
```diff
@@ -0,0 +1,171 @@
...
+}
+.message {
+  margin: 0 0 $spacer-extra-big 0;
+  font-size: $font-size-regular-mobile;
+  @include for-desktop {
+    font-size: $font-size-regular-desktop;
+  }
+  &__label {
+    font-weight: 400;
+  }
```
Comment (psmyrek) [inline components/molecules/m-update-personal-data.vue]: `.message__label` class is not used in template

File: components/molecules/m-update-password.vue (hunk 2 above)
Comment (qiqqq) [reply components/molecules/m-update-password.vue]: Fixed

File: components/molecules/m-update-personal-data.vue (hunk 3 above)
Comment (qiqqq) [reply components/molecules/m-update-personal-data.vue]: Fixed

File: components/molecules/m-update-password.vue (hunk 1 above)
Comment (qiqqq) [reply components/molecules/m-update-password.vue]: Fixed

**Pull request merged**